libs/libcommon/src/libcommon/config.py

```diff
@@ -318,6 +351,15 @@ class ProcessingGraphConfig:
                 ],
                 "job_runner_version": PROCESSING_STEP_DATASET_SPLIT_NAMES_VERSION,
             },
+            "split-descriptive-stats": {
+                "input_type": "split",
+                "triggered_by": [
+                    "config-split-names-from-info",
+                    "config-split-names-from-streaming",
+                    "config-parquet-and-info",
```

Comment:
why not just by "config-parquet-and-info" ?

Reply:
as for https://github.com/huggingface/datasets-server/pull/1441#discussion_r1244897544, maybe we can remove "config-parquet-and-info" to save resources (ie run the step a bit less often)